src/environmentd/tests/sql.rs

```diff
@@ -3343,3 +3344,58 @@ fn test_pg_cancel_backend() {
         "current transaction is aborted"
     );
 }
+
+// Test params in interesting places.
+#[mz_ore::test]
+fn test_params() {
```

Comment:
Should we add a `COPY` test too?

Reply:
COPY isn't supported in postgres so I'm reverting that part of the change.